### DISCUSSION TITLE
Is there an equivalent to keccak256(abi.encode(*some address*)) in web3j?

### DISCUSSION THREAD

**Discussion by andras-ballai** [web3j/web3j#1953] [Q&A] [answered]
I'm working with the t-rex token standard and it'd nice to have a java implementation of `keccak256(abi.encode(*some address*))` instead of having to create (and deploy) a smart contract just for this

**Comment by NickSneo** [comment, marked as answer]
You mean this - 

```
/**
     * Keccak-256 hash function.
     *
     * @param hexInput hex encoded input data with optional 0x prefix
     * @return hash value as hex encoded string
     */
    public static String sha3(String hexInput) {
        byte[] bytes = Numeric.hexStringToByteArray(hexInput);
        byte[] result = sha3(bytes);
        return Numeric.toHexString(result);
    }
```

This is in Hash Class - https://github.com/web3j/web3j/blob/master/utils/src/main/java/org/web3j/crypto/Hash.java
